token/program-2022/Cargo.toml

```diff
@@ -20,6 +20,7 @@ num-traits = "0.2"
 num_enum = "0.5.4"
 solana-program = "1.9.9"
 solana-zk-token-sdk = "0.6.0"
+spl-memo = { path = "../../memo/program", features = [ "no-entrypoint" ] }
```

Comment:
Will this need a version number to publish to crates.io?

Reply:
Probably! Thanks for catching, not sure why/how I missed that.